overlord/devicestate/devicemgr.go

```diff
@@ -2332,7 +2333,8 @@ func (m *DeviceManager) hasFDESetupHook(kernelInfo *snap.Info) (bool, error) {
 			return false, fmt.Errorf("cannot get kernel info: %v", err)
 		}
 	}
-	return hasFDESetupHookInKernel(kernelInfo), nil
+	_, ok := kernelInfo.Hooks["fde-setup"]
```

Comment:
It is a bit strange that this was previously a call to a function and now not anymore, when `hasFDESetupHookInKernel` is defined and used in overlord/install. Maybe it would be better to define too in overlord/devicestate for coherence.